src/patternfly/components/Nav/nav.scss

```diff
@@ -602,8 +573,7 @@
     top: 0;
     bottom: 0;
     content: "";
-    border-color: var(--pf-c-nav__scroll-button--before--BorderColor);
-    border-style: solid;
+    border: var(--pf-c-nav__scroll-button--before--BorderColor) solid;
```

Comment:
Can you list the color after the border-style? We maintain the shorthand order even with 2 values.

```
options-menu.scss:    border: var(--pf-c-options-menu__toggle--BorderWidth) solid;
context-selector.scss:    border: var(--pf-c-context-selector__toggle--BorderWidth) solid;
dropdown.scss:    border: var(--pf-c-dropdown__toggle--BorderWidth) solid;
input-group.scss:  border: var(--pf-c-input-group__text--BorderWidth) solid;
...
```